atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2123,21 +2113,18 @@ protected ConflictHandler getConflictHandlerForTable(TableReference tableRef) {
     /**
      * Make sure we have all the rows we are checking already locked before calling this.
      */
-    protected void throwIfConflictOnCommit(LockToken commitLocksToken, TransactionService transactionService)
-            throws TransactionConflictException {
+    protected void throwIfConflictOnCommit(TransactionService transactionService) throws TransactionConflictException {
```

Comment:
Only called from one place, so this usage must be correct, we'll end up sourcing the locks in the same way.

Reply:
Agreed.